lib/utils/__tests__/isNestedProperty.test.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+const isNestedProperty = require("../isNestedProperty");
+const postcss = require("postcss");
+
+describe("isNestedProperty", () => {
+  it("accepts nested property", () => {
+    return nestedProperty("foo: {};", nestedProperty => {
+      expect(isNestedProperty(nestedProperty)).toBeTruthy();
+    });
+  });
+
+  it("rejects nested property", () => {
```

Comment:
rejects **not** nested property

Reply:
But I have done that, no?! If `foo: {}`, is a nested property. If `foo: red`, isn't a nested property!

Pardon me, if I don't understand your request, I'm a little new in contributing to projects!

Reply:
I get it no! Consider done!

Reply:
test inside need describe what happens, i.e. `should be rejects not nested properties`

Reply:
@evilebottnawi Sorry, my misunderstanding! On a second read, I got what you were meaning!

Thanks very much for all the help!!